diff -Nru busybox-1.17.1.ORIG//Makefile.flags busybox-1.17.1/Makefile.flags
--- busybox-1.17.1.ORIG//Makefile.flags	2012-08-03 15:39:32.345043691 +0900
+++ busybox-1.17.1/Makefile.flags	2012-08-03 15:44:37.017043654 +0900
@@ -115,10 +115,6 @@
 LDLIBS += dmalloc
 endif
 
-ifeq ($(CONFIG_SYSLOGD),y)
-LDLIBS += systemd-daemon
-endif
-
 # If a flat binary should be built, CFLAGS_busybox="-elf2flt"
 # env var should be set for make invocation.
 # Here we check whether CFLAGS_busybox indeed contains that flag.
diff -Nru busybox-1.17.1.ORIG//sysklogd/syslogd.c busybox-1.17.1/sysklogd/syslogd.c
--- busybox-1.17.1.ORIG//sysklogd/syslogd.c	2012-08-03 15:39:32.349043691 +0900
+++ busybox-1.17.1/sysklogd/syslogd.c	2012-08-03 15:40:05.013043687 +0900
@@ -34,8 +34,6 @@
 #include <sys/shm.h>
 #endif
 
-#include "sd-daemon.h"
-
 #define DEBUG 0
 
 /* MARK code is not very useful, is bloat, and broken:
@@ -513,9 +511,6 @@
 	int sock_fd;
 	char *dev_log_name;
 
-	if (sd_listen_fds(1) == 1 && sd_is_socket_unix(SD_LISTEN_FDS_START, SOCK_DGRAM, -1, "/dev/log", 0) > 0)
-		return SD_LISTEN_FDS_START;
-
 	memset(&sunx, 0, sizeof(sunx));
 	sunx.sun_family = AF_UNIX;
 
